openedx/features/marketplace/constants.py

```diff
@@ -0,0 +1,33 @@
+ORGANIZATION_SECTOR_CHOICES = (
+    ('health', 'Health & Well-being'),
+    ('education', 'Education'),
+    ('equality', 'Gender Equality'),
+    ('sanitation', 'Sanitation'),
+    ('climate', 'Climate Change'),
+    ('energy', 'Clean Energy'),
+    ('environmental', 'Environmental Conservation'),
+    ('economic', 'Work & Economic Growth'),
+    ('rights', 'Human Rights'),
+    ('justice', 'Social Justice'),
+    ('art', 'Art & Culture')
+)
+
+ORGANIZATIONAL_PROBLEM_CHOICES = (
+    ('healthcare', 'Healthcare Supplies'),
+    ('remote-tools', 'Remote Working Tools'),
+    ('training', 'Online Training'),
+    ('delivery', 'Delivery Services'),
+    ('mentorship', 'Mentorship'),
+    ('funding', 'Funding'),
+    ('resources', 'Human Resources'),
+)
+
+USER_SERVICES = (
```

Comment:
ORGANIZATION_SERIVCES is more relevant here.

Reply:
@ahmed-faraz46 these are the services a user is offering, so I think it is more.

Reply:
My bad I misinterpreted it.